### PR TITLE
bem-info: Inline svg-code turns into a picture

### LINKED ISSUE #125: Inline svg inside bemjson snippet not escaped properly.
Inline svg in usage example of `icon` block not escaped on site and rendered as image.
See last example of bemjson on [`icon` block documentation page](https://ru.bem.info/libs/bem-components/v2.0.0/desktop/icon/)

### PR DESCRIPTION
**Pull request by gela-d**
Before:

After:

---

Close #125

### CODE REVIEW

**Comment by gela-d** [comment]
@tavriaforever

File: src/blocks/common.blocks/block-example/_view/block-example_view_inline.bemtree [hunk 1]
```diff
@@ -23,7 +23,7 @@ block('block-example').mod('view', 'inline')(
                         {
                             elem: 'source-code',
                             elemMods: { view: 'text' },
-                            content: this._bemjson
+                            content: this._bemjson.replace(/</g, '&lt;').replace(/>/g, '&gt;')
```
Comment (tadatuta) [inline src/blocks/common.blocks/block-example/_view/block-example_view_inline.bemtree]: I suggest to use [strings](https://ru.bem.info/libs/bem-core/v2.5.1/desktop/strings/) here.

**Comment by gela-d** [comment]
@tadatuta I found another way - https://github.com/bem/bem-site-engine/commit/5e1afe6f05f7b5bf1608c8a23d325f38ea54a0d3 Is it ok?

**Comment by tavriaforever** [comment]
> 5e1afe6

me OK this variant

**Comment by tadatuta** [comment]
yeah, sure! :+1:

**Pull request merged**